spring-cloud-gcp-starters/spring-cloud-gcp-starter-bigquery/pom.xml

```diff
@@ -12,6 +12,7 @@
   <artifactId>spring-cloud-gcp-starter-bigquery</artifactId>
   <name>Spring Cloud GCP BigQuery Starter</name>
   <description>Starter for Google Cloud BigQuery</description>
+  <url>https://github.com/spring-cloud/spring-cloud-gcp/tree/master/spring-cloud-gcp-starters/spring-cloud-gcp-starter-bigquery</url>
```

Comment:
If people go to this URL, they're not going to get any useful information. Would it make more sense to just link to the project hone page? This applies to other modules as well.

Reply:
I take this back since it won't be surfaced to users.